src/Components/allTodoFunctionality/removeAll.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+const RemoveAllTodo = ({ setCheckedStatus }) => {
+    // console.log(checkedStatus)
+    const removeList = () => {
+        setCheckedStatus([])
```

Comment:
Instead of using memory to creat a function you can add a ternary in the onClick that checks if checkedStatus length is greater then or equal to two if not then null

Reply:
make sure to use () => before the setCheckedStatus as this will only invoke the function if the button is clicked